.changeset/add-standards-mail-unsubscribe.md

```
@@ -0,0 +1,5 @@
+---
+'unwired-mail': patch
+---
+
+Detect unsubscribe headers for Standards-Based IMAP mail and route email requests through its durable SMTP Outbox.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)